File: PPA/Assignments/InputOutputAssignment2/Programs/question3.c
/*
What does the following C statement mean write detailed explanation?
scanf("%7s", str);
*/

when we use %7s in scanf it means that compiler only takes upto 7 characters as a input
 Eg:
 	if we have string as "World"
 	and scanf("%7s",str) then it will take input as whole -> w o r l d
 	when scanf("%4s",str) then it will take input as -> w o r l 
	when scanf("%2s",str) then it will take input as -> w o

File: Java/Assignments/OperatorAssignment/programs/question3.java
/*
what will be the output for following statement when a=10,and b=20,c=50
	a++ + ++b + ++c 
*/
class Question3{
	public static void main(String[] arg){
		int a=10,b=20,c=50,ans=0;
		ans=a++ + ++b + ++c;
		/*
			ans=a++ + ++b + ++c
			ans=10 + 21
			ans=31+51
			ans=82
		*/
		System.out.println("ans="+ans);	//82
	}
}
